### ISSUE TITLE
Request Body is always <none>

### ISSUE THREAD

**Issue by avaroam** [netguru/ResponseDetective#31]
I've setup ResponseDetective with Alamofire and it works great for responses. But the body of my requests in the console always shows as `<none>`, even when I am including parameters with the request.

```
 ├─ Headers
 │ Content-Type: application/x-www-form-urlencoded; charset=utf-8
 │ Content-Length: 18
 ├─ Body
 │ <none>
```

Is there a way to have it include the request parameters in the console log?

**Comment by akashivskyy** [comment]
It seems ResponseDetective doesn't recognize `application/x-www-form-urlencoded` content-type. Will address this asap.

**Comment by akashivskyy** [comment]
Released in [1.1](https://github.com/netguru/ResponseDetective/releases/tag/1.1).
